blogposts/2022/release-3.36.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Sourcegraph 3.36 release'
+publishDate: 2022-01-21T10:00-07:00
+description: 'Sourcegraph 3.36 introduces Notebooks and query-based search contexts in Beta, a new file fuzzy finder, and the ability to push branches from Batch Changes to forks.'
```

Comment:
The order of items in this description doesn't match the order in the post itself. Should we make them consistent? I'm assuming the order in the post itself was decided based on priority/impact of the items

Reply:
```suggestion
description: 'Sourcegraph 3.36 introduces the ability to push branches from Batch Changes to forks, a new file fuzzy finder, Notebooks for code search documentation, and query-based search contexts.'

Reply:
Good call! Re-working this based on the post order.